contributors/guide/issue-triage.md

```diff
@@ -4,57 +4,233 @@ weight: 10
 
-Sometimes users ask for support requests in issues; these are usually requests
-from people who need help configuring some aspect of Kubernetes. These issues
-should be labeled with `triage/support`, directed to our support structures
-(see below) and then closed. Also, if the issue is clearly abandoned or in the
-wrong place, it should be closed. Keep in mind that only issue reporters,
-assignees and component organization members can close issues. If you do not
-have such privilege, just comment your findings. Otherwise, first `/assign`
-issue to yourself and then `/close`.
+We suggest preparing your triage by filtering out the oldest, unlabelled issues and/or pull requests first.
+
+## Step Two: Triage Issues by Type
+Use [these labels](https://github.com/kubernetes/kubernetes/labels?utf8=%E2%9C%93&q=triage%2F+kind%2Fsupport) to find open issues that can be quickly closed. A triage engineer can add the appropriate labels.
+
+Depending on your permissions, either close or comment on any issues that are identified as support requests, duplicates, or not-reproducible bugs, or that lack enough information from the reporter.
+ 
+### Support Requests
+Some people mistakenly use GitHub issues to file support requests. Usually they're asking for help configuring some aspect of Kubernetes. Here's how to handle these:
+* First, direct the person to discuss the issue on [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes) and visit the [troubleshooting guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/)
+* Then, apply the `triage/support` label, which is directed to our support structures (see below), and apply the `close` label
+
+Please find more detailed information about Support Requests in the [Footnotes section](#footnotes).
+
+### Duplicates
+* Duplicates of other open issues should be left with a comment referencing the original issue and closed.
 
-### Support Structures
+### Abandoned or Wrongly Placed Issues
+Depending on your permissions, either close or comment on it.
 
-Support requests should be directed to the following:
+### Needs More Information
+* The `triage/needs-information` label indicates an issue needs more information in order to work on it; comment on or close it.
 
+### Bugs
+First, validate if the problem is a bug by trying to reproduce it.
+
+If you can reproduce it:
+* [Define its priority](##step-three-define-priority)
+* Do a quick duplicate search to see if the issue has been reported already. If a duplicate is found, let the issue reporter know, reference the original issue, and close the duplicate.
+
+If you can't reproduce it:
+* Contact the issue reporter with your findings 
+* Close the issue if both the parties agree that it could not be reproduced.
+
+If you need more information to further work on the issue:
+* let the reporter know it by adding an issue comment followed by label `lifecycle/needs-information`.
```

Comment:
nit, few places in the doc like this one here, there is inconsistency in how the sentences is started i.e. with a capital or small letter. e.g. "let the reporter know.." vs "Contact the issue reporter.."